docs/quickstart/invoice_model.mdx

```diff
@@ -58,18 +48,24 @@ To train a model, **the field names always need to match the labels** in the dat
 </TabItem>
 </Tabs>
 
-3. Go to [**Datasets**](https://app.cradl.ai/datasets) > [**New dataset**](https://app.cradl.ai/datasets/new), give it a name and press **Create**.
-4. Go to [**Datasets**](https://app.cradl.ai/datasets) > YourDataset > Upload data and upload all the files from the [sample invoices](/sample_invoices.zip).
+## 2. Configure your model
+We are now ready to enter Cradl and put our data to use.
+1. In Cradl, go to [**Models**](https://app.cradl.ai/models) > [**New model**](https://app.cradl.ai/models/new).
+2. Give your model a name, and add an optional description.
+3. Choose the invoice template and press **Create model**.
+4. In the *Model overview* section press **Go to training** and then **New training**.
+
+:::info Matching fields
+To train a model, **the field names always need to match the labels** in the data.
+:::
 
 ## 3. Train your model
-To make sure that your dataset is compatible with the model we will have to create a [data bundle](../concepts/training-data.mdx).
-
-1. Go to **[Models](https://app.cradl.ai/models) > YourModel > Data bundles > Select Data**.
-2. Give your data bundle a name, and add an optional description.
-3. Choose the dataset you want to train on.
-4. Press **Create Data Bundle** and wait for the data bundle summary to appear.
-5. Inspect the summary to make sure that the training data is of sufficiently good quality to start training.
-6. Go to **[Models](https://app.cradl.ai/models) > YourModel > Training > New training** and choose the data bundle you created in the previous step.
+You should now be in the training wizard, if not just go to
+[**Models**](https://app.cradl.ai/models) > **Your model** > **Training jobs** > **New training**.
+1. Choose **Create new dataset**, give your dataset a name and add an optional description.
+2. Choose all the files you prepared in the first step and press start upload, remember to add both JSONs and PDFs.
+3. Generate a data report, this step will be more important when dealing with real data.
+4. Leave the default settings and press **Start model training** to send a training request.
```

Comment:
Maybe add a note about what the user can expect in regards of training and approval time